.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,27 @@
+name: Precommit Python Formatter
+run-name: Python Formatter
+on:
```

Comment:
I think we should also trigger this on pushes to master:

```
push:
    tags: ['v*']
    branches: ['master', 'release-*']
```

This applies elsewhere in this PR (and actually, we missed this trigger and the comment trigger in https://github.com/apache/beam/blob/master/.github/workflows/beam_PreCommit_Go.yml